lib/Pod/To/Text.pm6

```diff
@@ -33,14 +33,32 @@ sub pod2text($pod) is export {
 
 sub heading2text($pod) {
     given $pod.level {
-        when 1  {          pod2text($pod.contents)  }
-        when 2  { '  '   ~ pod2text($pod.contents)  }
+        my $content = $pod.contents.first.contents.grep( * !~~ "").first;
+        my $text;
+        if $content ~~ Str {
+            $text = $content;
+        } else {
+            $text = pod2text $content;
+        }
+        my $name = $text.lines[0];
+        my $rest = $text.lines[1…*].join: "\n";
+        when 1  {
+            my $result = colored($name.uc, 'bold') ~ "\n";
+            $result ~= "\n$rest" if $rest.defined;
+            $result.trim;
+        }
+        when 2  {
+            my $result = '  ' ~ colored($name, 'bold') ~ "\n";
+            $result ~= "\n$rest" if $rest.defined;
+            $result.trim-trailing;
+        }
         default { '    ' ~ pod2text($pod.contents)  }
     }
 }
 
 sub code2text($pod) {
-    "    " ~ $pod.contents>>.&pod2text.subst(/\n/, "\n    ", :g)
+    my $text = "    " ~ $pod.contents>>.&pod2text.subst(/\n/, "\n    ", :g);
```

Comment:
The only case where they won't do the same thing is if there is a trailing "\n", in which case what you are doing could potentially cause unwanted indentation happening.
```perl6
("    " ~ "$var\n".subst(/\n/, "\n    ", :g)).perl
# "    here is some\n    text right here\n    rigth here\n    right there\n    "
```
Compare this to:
```perl6
"$var\n".indent(4).perl
# "    here is some\n    text right here\n    rigth here\n    right there\n"
```